index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>My portafolio</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="css/style.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+  <nav class="toolbar">
+    <p id="toolbar-name">Richard Burneo</p>
+    <i id="hamburguer" class="fa fa-bars" aria-hidden="true"></i>
+  </nav>
```

Comment:
- [ ] For this section, kindly wrap in round a `header` tag. This is in line with best practices when sectioning your navigation basically 👍